From
http://sources.redhat.com/cgi-bin/cvsweb.cgi/libc/po/Makefile.diff?r1=1.17&r2=1.18&cvsroot=glibc

Fixes error
  msgfmt -o el.mo el.po
  el.po:538:31: invalid multibyte sequence
  el.po:539:21: invalid multibyte sequence
  el.po:1537:38: invalid multibyte sequence
  msgfmt: found 3 fatal errors
  make[2]: *** [el.mo] Error 1
  make[2]: Leaving directory
  `/home/dkegel/wk/crosstool-0.25/build/i686-unknown-linux-gnu/gcc-2.95.3-glibc-2.1.3/glibc-2.1.3/po'

===================================================================
RCS file: /cvs/glibc/libc/po/Makefile,v
retrieving revision 1.17
retrieving revision 1.18
diff -u -r1.17 -r1.18
--- libc/po/Makefile	1999/09/13 08:48:40	1.17
+++ libc/po/Makefile	2001/04/20 04:45:54	1.18
@@ -1,4 +1,4 @@
-# Copyright (C) 1996, 1997, 1998 Free Software Foundation, Inc.
+# Copyright (C) 1996, 1997, 1998, 1999, 2001 Free Software Foundation, Inc.
 # This file is part of the GNU C Library.
 
 # The GNU C Library is free software; you can redistribute it and/or
@@ -20,8 +20,12 @@
 
 subdir := po
 
+# List of languages that are currently broken (just run msgfmt from
+# GNU gettext 0.10.36 on them to see why)
+BROKEN_LINGUAS = el
+
 # List of languages for which we have message catalogs of translations.
-ALL_LINGUAS := $(basename $(wildcard *.po))
+ALL_LINGUAS := $(filter-out $(BROKEN_LINGUAS),$(basename $(wildcard *.po)))
 
 # You can override this in configparms or the make command line to limit
 # the languages which get installed.
